src/pages/QuestionPage.tsx

```diff
@@ -74,6 +77,8 @@ function QuestionPage() {
   }
 
   const checkAnswer = (event: FormEvent) => {
+    const successAudio = new Audio(SuccessAudio)
```

Comment:
consider using a 3rd party library like [Howler](https://github.com/goldfire/howler.js) to manage the audio

and set the playing volume around 0.5 to prevent startled user

Reply:
create some helper in utils like this

```ts
import { Howl } from 'howler';

const sounds = {
  success: new Howl({
    src: ['file'],
    volume: 0.5,
  }),
};

const playSound = (sound) => {
  if (sounds[sound]) {
    sounds[sound].play();
  }
};

export default playSound;
```